### DISCUSSION TITLE
the consistency of photometry data

### DISCUSSION THREAD

**Discussion by KGadallah** [mboquien/cigale#18] [General]
How can we check the consistency of photometry in the optical-to-FIR?

**Comment by Guang91** [comment]
Could you be more specific? What do you mean by consistency?

**Reply by KGadallah** [reply]
When we compile the flux densities from the NED databases which may be quite heterogeneous, particularly, the MIR-FIR flux densities for high-z sources. Sometimes, we need to consider the contamination from the neighborhood because the spatial resolution of MIR-FIR is not so good enough compared with optical/NIR data. If the FIR flux of an object is affected by that from nearby sources, its MIR and FIR may be over-estimated. So, shall we need to double check the consistency of photometry data the optical-to-FIR to avoid the overestimation?

**Reply by Guang91** [reply]
I see what you mean. I'd suggest checking the SED (you can plot with "pcigale-plots sed"). If it's a mismatch, you would see either MIR/FIR or optical/NIR is systematically offset from the model (the model always keeps energy balanced, so it's physical).
